pkg/agent/dozer/bcm/plan.go

```diff
@@ -468,15 +531,48 @@ func planFabricConnections(agent *agentapi.Agent, spec *dozer.Spec) error {
 			}
 
 			spec.VRFs[VRFDefault].BGP.Neighbors[ip.String()] = &dozer.SpecVRFBGPNeighbor{
-				Enabled:                 pointer.To(true),
-				Description:             pointer.To(fmt.Sprintf("Fabric %s %s", remote, connName)),
-				RemoteAS:                pointer.To(peerSw.ASN),
-				IPv4Unicast:             pointer.To(true),
-				L2VPNEVPN:               pointer.To(true),
-				L2VPNEVPNImportPolicies: []string{RouteMapBlockEVPNDefaultRemote},
-				// TODO: We might later specify dedicated neighbors for this.
-				L2VPNEVPNAllowOwnAS: pointer.To(true),
-				BFDProfile:          pointer.To(FabricBFDProfile),
+				Enabled:                   pointer.To(true),
+				Description:               pointer.To(fmt.Sprintf("Fabric %s %s", remote, connName)),
+				RemoteAS:                  pointer.To(peerSw.ASN),
+				IPv4Unicast:               pointer.To(true),
+				IPv4UnicastExportPolicies: []string{RouteMapProtocolLoopbackOnly},
+				BFDProfile:                pointer.To(FabricBFDProfile),
+			}
+		}
+	}
+	// add the ebgp sessions over the protocol loopback of the neighbors
+	ownProtocolIP, _, err := net.ParseCIDR(agent.Spec.Switch.ProtocolIP)
+	if err != nil {
+		return errors.Wrapf(err, "failed to parse own protocol IP %s", agent.Spec.Switch.ProtocolIP)
+	}
+	ownProtocolIPStr := ownProtocolIP.String()
+
+	for peer := range peers {
+		peerSpec, ok := agent.Spec.Switches[peer]
+		if !ok {
+			return errors.Errorf("no switch found for peer %s", peer)
+		}
+		if peerSpec.ProtocolIP == "" {
+			return errors.Errorf("no protocol IP found for peer %s", peer)
+		}
+		ip, _, err := net.ParseCIDR(peerSpec.ProtocolIP)
+		if err != nil {
+			return errors.Wrapf(err, "failed to parse protocol IP %s for peer %s", peerSpec.ProtocolIP, peer)
+		}
+		ipStr := ip.String()
+		allowasIn := agent.Spec.Switch.Redundancy.Type == meta.RedundancyTypeMCLAG
+		if _, exists := spec.VRFs[VRFDefault].BGP.Neighbors[ipStr]; !exists {
```

Comment:
It can already exist only if we're facing a switching with the same protocol IP which shouldn't be possible but even if it is the neighbor config is exactly the same so no need to do a check